public/docs/dart/latest/tutorial/toh-pt6.jade

```diff
@@ -1 +1,139 @@
-!= partial("../../../_includes/_ts-temp")
+extends ../../../ts/latest/tutorial/toh-pt6.jade
+
+block includes
+  include ../_util-fns
+  - var _Http = 'BrowserClient';
+  - var _Angular_Http = 'Dart <code>BrowserClient</code>'
+  - var _httpUrl = 'https://pub.dartlang.org/packages/http'
+  - var _Angular_http_library = 'Dart <a href="' + _httpUrl + '"><b>http</b></a> library'
+  - var _HTTP_PROVIDERS = 'BrowserClient'
+  - var _JSON_stringify = 'JSON.encode'
+
+block start-server-and-watch
+  :marked
+    ### Keep the app compiling and running
+    Open a terminal/console window.
+    Start the Dart compiler, watch for changes, and start our server by entering the command:
+
+  code-example(language="bash").
+      pub serve
+ 
+block http-library
+  :marked
+    We'll be using the !{_Angular_Http} to communicate with a server.
+    This class is exported by the !{_Angular_http_library}.
+
+    ### Pubspec updates
+    
+    We need to add a package dependency for the !{_Angular_http_library}.
+
+    We also need to inform the Dart Angular 2 Transformer that we will be using
+    `BrowserClient` (we'll defer explaining _why_ this extra configuration is
```

Comment:
Move the "(we'll...)" to be after the end of the sentence. Also, let's phrase it a little differently so it's not about _us_ as explainers. Maybe:

We also need to add a resolved_identifiers entry, to inform the angular2 transformer that we'll be using BrowserClient. (For an explanation of why this extra configuration is needed, see the HTTP client chapter.) We'll also need to use Client from http, so let's add that now as well.